### PR TITLE
Process room topic event

### PR DESCRIPTION
**Pull request by pbruna**
This set the attribute `topic` for the Room.
Signed-off-by: Patricio Bruna <pbruna@zboxapp.com>

### CODE REVIEW

**Comment by richvdh** [comment]
Thanks for this, and really sorry for sitting on it for so long.

It looks great, but could you rebase it on latest develop?

**Comment by pbruna** [comment]
Hi @richvdh ,
Rebased.

**Comment by richvdh** [comment]
Thanks. could you fix the lint failures?

**Comment by pbruna** [comment]
Hi,
Lint fixed, but I don't know why the test fails on travis.

**Comment by richvdh** [comment]
I should have been more specific: please ensure that new code doesn't introduce *new* lint failures. In current develop there are 0 errors and 115 warnings:

```
npm run lint
...
✖ 115 problems (0 errors, 115 warnings)
```

Fixes to existing warnings are welcome, but please don't do it in existing code in the same PR as adding new functionality.

> Lint fixed, but I don't know why the test fails on travis.

well, it looks like you've broken the tests. Can you not reproduce the test failure locally?

File: src/models/room-state.js [hunk 1]
```diff
@@ -182,7 +182,7 @@ RoomState.prototype.setStateEvents = function(stateEvents) {
             utils.forEach([member, sentinel], function(roomMember) {
                 roomMember.setMembershipEvent(event, self);
                 // this member may have a power level already, so set it.
-                const pwrLvlEvent = self.getStateEvents("m.room.power_levels", "");
+                const pwrLvlEvent = self.getStateEvents("m.room.powerLevels", "");
```
Comment (richvdh) [inline src/models/room-state.js]: apart from my request not to make unrelated changes in the same PR, this looks wrong.
